src/Altinn.App.Core/Internal/Patch/PatchService.cs

```diff
@@ -202,6 +203,24 @@ await RunDataProcessors(
         };
     }
 
+    public async Task<List<ValidationSourcePair>> RunIncrementalValidation(
```

Comment:
What's the rationale for adding this?

Reply:
Just that I did not want to inject the full ValidationService twice for PatchService and directly in the controller.

Reply:
Would prefer to just inject another service tbh 😄

Reply:
Yes, when we get your patch to dynamically initialise the validators, so that it don't happen twice for every request (service owners does strange things in their constructors), it would be my preference too.

Actually I would prefer to not have PatchService in the first place, because it doesn't have natural boundaries, but it was added in code review to make DataController shorter 🥲.